tools/build/lib/byond.js

```diff
@@ -85,6 +86,14 @@ export const DreamMaker = async (dmeFile, options = {}) => {
   };
   testOutputFile(`${dmeBaseName}.dmb`);
   testOutputFile(`${dmeBaseName}.rsc`);
+  var runWithWarningChecks = async (dmeFile, error, options = {}) => {
```

Comment:
const

Reply:
Done. I've noticed I didn't pass arguments to Juke.exec correctly, I've fixed it too.